[DX-1781] Release notes for 5.3.9 (#5826)
TykTechnologies/tyk-docs: tyk-docs/content/developer-support/release-notes/dashboard.md

## 5.3 Release Notes

> ### 5.3.9 Release Notes

Reviewer: ```suggestion

### 5.3.9 Release Notes
```

## 5.3 Release Notes

### 5.3.9 Release Notes

#### Release Date xxx

#### Release Highlights

> This release focuses mainly on bug fixes. For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.3.9">}}) below.

Reviewer: ```suggestion
This release contains a bug fix. For a comprehensive list of changes, please refer to the detailed [changelog]({{< ref "#Changelog-v5.3.9">}}) below.
```

- Link to documentation of the new feature
- For OSS - Link to the corresponding issue if possible on GitHub to allow the users to see further info.

Each change log item should be expandable. The first line summarises the changelog entry. It should be then possible to expand this to reveal further details about the changelog item. This is achieved using HTML as shown in the example below. -->

<ul>
<li>
<details>
<summary>Fixed Issue with Restore Zooming in API Activity Dashboard</summary>

Resolved a bug where clicking "Restore zooming to initial state" in the API Activity Dashboard would cause the graph to show blank instead of resetting to the initial zoom level. This fix ensures that users can now correctly restore the default zoom state in all charts on the Dashboard.
</details>
</li>
<li>
<details>
<summary>Deprecation of http_server_options.prefer_server_ciphers</summary>

This option has been marked as deprecated due to its obsolescence in the underlying library functions used by Tyk. Users are advised to remove this configuration from their setups as it no longer has any effect.
</details>
</li>
<li>
<details>
<summary>CVE-2020-8911 resolved in Tyk Dashboard</summary>

> Resolved CVE-2020-8911 by updating the Tyk Dashboard's email driver to use AWS SDK v2, addressing a medium-severity security vulnerability identified in version 5.3.8. This update ensures enhanced security for the dashboard while maintaining functionality.

Reviewer 2: ```suggestion
Resolved CVE-2020-8911 by updating the Tyk Dashboard's email driver to use AWS SDK v2, addressing a medium-severity security vulnerability identified in version 5.3.8. This update ensures enhanced security for the Dashboard while maintaining functionality.
```

Resolved an issue where API access keys remained valid even if all associated policies were deleted. The Gateway now attempts to apply all linked policies to the key when it is presented with a request. Warning logs are generated if any policies cannot be applied (for example, if they are missing). If no linked policy can be applied, the Gateway will reject the key to ensure no unauthorized access.
</details>
</li>
<li>
<details>
<summary>Deprecation of http_server_options.prefer_server_ciphers</summary>

This option has been marked as deprecated due to its obsolescence in the underlying library functions used by Tyk. Users are advised to remove this configuration from their setups as it no longer has any effect.
</details>
</li>
<li>
<details>
<summary>Improved Stability for APIs with Malformed Listen Paths</summary>

Fixed an issue where a malformed listen path could cause the Gateway to crash. Now, such listen paths are properly validated, and if validation fails, an error is logged, and the API is skipped—preventing Gateway instability.
</details>
</li>
<li>
<details>
<summary>CVE-2020-8911 resolved in Tyk Dashboard</summary>

> This option has been marked as deprecated due to its obsolescence in the underlying library functions used by Tyk. Users are advised to remove this configuration from their setups as it no longer has any effect.

Reviewer 2: This description is wrong - it's a copy/paste from above, doesn't relate to this fix.
